cluster/src/test/java/io/scalecube/cluster/fdetector/FailureDetectorIT.java

```diff
@@ -461,4 +459,13 @@ private void assertStatus(
       throw new RuntimeException(e);
     }
   }
+  private <T> CompletableFuture<List<T>> allOf(List<CompletableFuture<T>> futuresList) {
+    CompletableFuture<Void> allFuturesResult =
+            CompletableFuture.allOf(futuresList.toArray(new CompletableFuture[futuresList.size()]));
+    return allFuturesResult.thenApply(v ->
```

Comment:
Yes, I was thinking about some utility interface with static methods like Futures. But it can be done later, when needed in more places.